fvtest/compilertest/tests/forwardListTest.cpp

```diff
@@ -0,0 +1,84 @@
+/****************************************************************************
+ Copyright IBM Corp. 2017, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+*******************************************************************************/
+
+
+#include "infra/forward_list.hpp"
+#include "env/TRMemory.hpp"
+#include <string>
+#include "gtest/gtest.h"
+using namespace std;
+
+
+class forward_list_test : public :: testing :: Test {
+
+        protected:
+
+                TR:: forward_list <int, allocator<int>> list1;
```

Comment:
Please use `int32_t` where you have `int` in this test.  This doesn't just apply to this line, but everywhere in this test.